spec/acceptance/actions_spec.rb

```diff
@@ -25,6 +25,7 @@ class { 'rsyslog::config':
           'myaction' => {
             'type'   => 'omelasticsearch',
             'config' => {
+              'host'                 => '["127.0.0.1", "192.168.0.1"]',
```

Comment:
Is there a reason why the array is actually a string? can't we use a real array datatype here?

Reply:
No reason. It works with array too.